services/cd-service/pkg/repository/transformer_db_test.go

```diff
@@ -1289,6 +1289,49 @@ func TestEvents(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name: "Replaced By test",
+			// no need to bother with environments here
```

Comment:
not sure what this comments means. Is the `CreateEnvironment` transformer still needed here?

Reply:
We do need to bother, so that the CreateApplicationVersion actually deploys it. Copy-paste mistake from previous tests